edgedb-init.sh

```diff
@@ -0,0 +1,269 @@
+#!/bin/bash
+
+# This is just a little script that can be downloaded from the internet to
+# install eddgedb CLI tools. It just does platform detection, downloads the
+# installer and runs it.
+
+set -u
+
+EDGEDB_PKG_ROOT="${EDGEDB_PKG_ROOT:-https://packages.edgedb.com}"
+
+usage() {
+    cat 1>&2 <<EOF
+edgedb-init
+The installer for edgedb command-line tools
```

Comment:
```suggestion
The installer for edgedb CLI.
```

Reply:
Why? I see general attitude to avoid abbreviations in open-source community. We're not limited in the space here.

Reply:
Simply because I used "CLI" in other suggested edit for the same file. You can disregard this, the current text works too.